### PR TITLE
Version info fix

### PR DESCRIPTION
**Pull request by saba8814**

### CODE REVIEW

File: simple_netconf_client/__init__.py [hunk 1]
```diff
@@ -34,3 +23,14 @@ def setup_logging(debug_enabled):
...
 
     logging.basicConfig(level=logging_level, format='%(asctime)s - %(levelname)s - %(message)s')
+
+try:
+    version_file_path = resource_path('version.txt')
+    with open(version_file_path) as f:
+        __version__ = f.readlines()[0].strip()
+except Exception:
+    # package is not installed
+    __version__ = "0.0.0"
```
Comment (troglobit) [inline simple_netconf_client/__init__.py]: Running the program from a `git clone` with `python main.py` now shows version '0.0.0'.  I would've expected 'v2.0.0-8-g6de3873' == 8 commits after tag 'v2.0.0', specifically git has 6de3873.

Tricky.  We don't need 'import git' at runtime for people running the .exe.  Maybe we can keep the code in `__init__.py` and only call it if we don't have `version.txt` perhaps?

File: simple_netconf_client/gui/Dialogs.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def __init__(self, parent, width, height):
         parent.center_dialog(self, width, height)
         about_message = (
             "Simple NETCONF Client\n"
-            f"v{get_version()}\n"
+            f"{get_version()}\n"
```
Comment (troglobit) [inline simple_netconf_client/gui/Dialogs.py:15]: Found the root cause of this.  The previous tag did not have a leading `v`.  I suggest we stick to one scheme in the future for consistency :-)

If you like we can add a `v1.0.0` tag at the same git hash.

**Review by troglobit — APPROVED** [review]
My review comment is more of a "would be nice if worked in both cases", if you're up for the challenge, supporting both use-cases would be great.  

In any case, it still seems to work fine on Linux, even connecting to and managing an Infix device, so I'm still a happy camper :-)

File: simple_netconf_client/__init__.py (hunk 1 above)
Comment (saba8814) [reply simple_netconf_client/__init__.py]: I've pushed new commit addressing this issue. You can review.

Comment (troglobit) [reply simple_netconf_client/__init__.py]: Thanks, looks good to me! :sunglasses::+1: 

Appreciate you remember updating `requirements.txt` as well :pray:

**Pull request merged**